File: Sheet #1 (Data type - Conditions)/P_First_digit.cpp
#include <bits/stdc++.h>

using namespace std;
int main()
{

    int a;
    cin >> a;

    while (a >= 10)
        a /= 10;

    // cout << a;

    if (a % 2 == 1)
        cout << "ODD";
    else
        cout << "EVEN";

    return 0;
}

File: Sheet #1 (Data type - Conditions)/P_First_digit.java
import java.util.*;
import java.lang.*;
import java.io.*;

// The main method must be in a class named "Main"
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int n = scan.nextInt();

        while(n>=10)
          n/=10;

        if(n%2==0)
          System.out.println("EVEN");
        else
          System.out.println("ODD");    
        

    }
}
